build.jl

```diff
@@ -7,47 +11,61 @@ f = readdir("images/")
 # Read yaml metadata
 imageMetadata = YAML.load(open("metadata.yml"))
 
-# Initialize Thumbnails directory and set default height to 200
-if isdir("thumbnails")
-  rm("thumbnails/", recursive=true)
+# Initialize Thumbnails directory if it does not yet exist
+if ~isdir("thumbnails")
+  println("Creating thumbnails/ folder.")
+  mkdir("thumbnails")
+end
+# copy default if not yet copied
+if ~isfile("thumbnails/cannot_be_displayed.png")
```

Comment:
```suggestion
if !isfile("thumbnails/cannot_be_displayed.png")
```

Reply:
Uh, my old Matlab history might have disturbed me here. Thanks for noticing.